src/main/java/com/godaddy/sonar/ruby/simplecovrcov/MockedCoverageSettings.java

```diff
@@ -0,0 +1,39 @@
+package com.godaddy.sonar.ruby.simplecovrcov;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Created by sergio on 3/27/17.
+ */
+public class MockedCoverageSettings implements CoverageSettings {
```

Comment:
this class is only used for testing purposes, shouldn't it be at the tests directory?